ksql-engine/src/main/java/io/confluent/ksql/codegen/CodeGenRunner.java

```diff
@@ -211,6 +212,13 @@ protected Object visitArithmeticBinary(
       return null;
     }
 
+    protected Object visitArithmeticUnary(
```

Comment:
How come this method didn't exist previously? Is this fixing a bug or is it only needed for decimals?

Reply:
yes - previously all numerics were terminal primitive types. that is no longer necessarily the case